src/lib/gundb/UserStorageClass.js

```diff
@@ -2095,4 +2097,67 @@ export class UserStorage {
     logger.debug('deleteAccount', { deleteResults })
     return true
   }
+
+  async test(date) {
+    const lastBlock = await userStorage.getLastBlockNode().then()
+
+    goodWallet.listenTxUpdates(parseInt(lastBlock), ({ toBlock }) =>
+      userStorage.saveLastBlockNumber(parseInt(toBlock) + 1)
+    )
+  }
+
+  async getFeedSince(date) {
```

Comment:
this is overcomplicating things, i've asked to use getfeedpage, if last item received is still not seen then you get the next page and so on

Reply:
I'm not sure if I understand your suggestion. How you get the next page with “getFeedPage”, From what I've seen this receive a quantity of result like a param and not a feed page. I applied change with a fixed number of results.